pkg/storage/secret/metadata/decrypt_store.go

```diff
@@ -33,14 +34,14 @@ func ProvideDecryptStorage(
 		return nil, fmt.Errorf("failed to get keepers: %w", err)
 	}
 
-	return &decryptStorage{db: db, keepers: keepers, secureValueMetadataStorage: secureValueMetadataStorage, allowList: allowList}, nil
+	return &decryptStorage{db: db, keeperMetadataStorage: keeperMetadataStorage, keepers: keepers, secureValueMetadataStorage: secureValueMetadataStorage, allowList: allowList}, nil
 }
 
 // decryptStorage is the actual implementation of the decrypt storage.
 type decryptStorage struct {
 	// TODO: remove this and only use the storages (when making this a service), need to figure out the getKeeperConfig part.
-	db db.DB
-
+	db                         db.DB
```

Comment:
I think we can remove this now then no? 🤔

Reply:
Removed 👍